### ISSUE TITLE
(Test report) Dashboard inverts colors when I enter "Cthụ̵̀l̴̨̈́hu R̸͚͝'ly̸̝̓é̴̹h̴͚̚" as part of URL fragment

### ISSUE THREAD

**Issue by moxious** [zuchka/grafana-action-test#64]
### Grafana Version?

9.4.0-beta1

### Grafana OS?

Debian

### Grafana Platform?

A package manager (APT, YUM, BREW, etc.)

### User's OS?

MacOS

### User's Browser?

Chrome

### Is this a Regression?

Yes

### Are Datasources involved?

All

### What happened?

**What happened**:

(Something something)

**What did you expect to happen**:

(Elaborate expectations go here)

### Reproduction Steps?

**Step 1**:  Go to `http://localhost:3000`

**Step 2**: Enter in ?q=Cthụ̵̀l̴̨̈́hu R̸͚͝'ly̸̝̓é̴̹h̴͚̚ in the URL

**Step 3**: craziness ensues

### Is the bug in a Dashboard Panel?

Copy the panel's ["get-help" data](https://grafana.com/docs/grafana/latest/troubleshooting/send-panel-to-grafana-support/) here

N/A

### Relevant log output?

```shell
Copy your grafana server logs here
```

**Comment by zuchka** [comment]
@moxious Hi there! It looks like you forgot to add any labels to this issue. We ask that members of the Grafana Org avoid filing empty issues because it slows down our triage process. Please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one descriptive label AND add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one descriptive label and the triage folks will help place it with the correct team. This really helps us move as fast as possible. :rocket:

And please keep the `internal` tag so we can track issues created by grafanistas. Thank you! :heart:

**Comment by moxious** [comment]
(I think) automated @zuchka I can't add labels here, I guess for permissions reasons?  This follow on message isn't something I can act on (but we're just testing here)

**Comment by zuchka** [comment]
yep--please ignore that message! I'll turn off that action test rn
